integration-cli/docker_cli_swarm_test.go

```diff
@@ -835,3 +835,16 @@ func (s *DockerSwarmSuite) TestDNSConfigUpdate(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(strings.TrimSpace(out), checker.Equals, "{[1.2.3.4] [example.com] [timeout:3]}")
 }
+
+func (s *DockerSwarmSuite) TestServicePsFormat(c *check.C) {
+        d := s.AddDaemon(c, true, true)
```

Comment:
I don't think we need an integration test for this. It should be possible to unit test.